e2e/view-contact.e2e-spec.ts

```diff
@@ -0,0 +1,69 @@
+import { ContactListPageObject, ViewContactPageObject } from './po';
+
+fdescribe('viewing a contact', () => {
```

Comment:
did you mean to leave this as `fdescribe`?

Reply:
Good catch. I sometimes forget to turn those back on.